app/src/main/java/com/sopt/shinmungo/presentation/report/component/CategoryGrid.kt

```diff
@@ -0,0 +1,86 @@
+package com.sopt.shinmungo.presentation.report.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.GridItemSpan
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.sopt.shinmungo.core.designsystem.component.button.RoundedCornerTextButton
+import com.sopt.shinmungo.core.designsystem.theme.ShinMunGoTheme
+import com.sopt.shinmungo.presentation.report.ReportViewModel
+
+@Composable
+fun CategoryGrid(viewModel: ReportViewModel, modifier: Modifier = Modifier) {
```

Comment:
앞서 뷰모델을 파라미터로 받지 않는 것을 추천한다고 했는데, 다른 파라미터들을 받는 예시를 보여드릴게요!
```suggestion
fun CategoryGrid(
      illegalParkingCategory: List<String>,
      updateSelectedCategory: (String) -> Unit,
      updateIsDropDownOpen: () -> Unit,
      modifier: Modifier = Modifier
) {
```